diff --git a/src/Federation/Minisign.php b/src/Federation/Minisign.php
index 3ce2565..8e9534e 100644
--- a/src/Federation/Minisign.php
+++ b/src/Federation/Minisign.php
@@ -59,7 +59,7 @@ class Minisign
         // check whether we have a public key with this key ID
         $publicKey = self::getPublicKey($signatureKeyId, $encodedPublicKeyList);
 
-        return sodium_crypto_sign_verify_detached(
+        return \Sodium\crypto_sign_verify_detached(
             substr($msgSig, \strlen(self::SIGNIFY_ALGO_DESCRIPTION) + self::SIGNIFY_KEY_ID_LENGTH),
             $messageText,
             $publicKey
